metadata/overlays/google-cloudsql/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+generatorOptions:
+  # name suffix hash is not propagated correctly to base resources
```

Comment:
How come you are disabling the suffix name hash? This prevents a unique hash from being added to the config map based on the content.

Including the suffix hash is preferred because then the deployment will change if the content is changed and this will trigger a pods update.

Otherwise it can be difficult to tell whether a deployment is referring to the latest version of the config.

Reply:
Agree it is a good practice to have unique suffix to indicate the version of a configMap, but the generated configmap is not properly written into the metadata-deployment.yaml file if namesuffix is enabled. (See https://github.com/kubernetes-sigs/kustomize/issues/1301)

I updated the comment in the config to include the issue link.